coralproject/talk
# Admin actions for Flagged comments

## What does this PR do?
- Adds Admin actions for Flagged comments
- Changes `null` for `NONE`
- Solves the Flag Queue issue. It returns only `NONE` or `PREMOD` statuses for flagged comments.
- Old routes, and old actions clean up. (`/comments` endpoint free)

## How do I test this PR? 

- Flag a comment
- Go to Moderation 
- Check the actions for flagged comments
